src/pages/volunteer page/components/Filter/Checkbox.js

```diff
@@ -0,0 +1,78 @@
+import React, { useRef, useLayoutEffect, useState } from "react";
+import { status } from "./constants";
+import { data } from "./regionsData";
+
+export default function Checkbox({
```

Comment:
please be consistent with function declaration , i saw in. other places u have :

function YourComponent {}
export default YourComponent

and here u used :
export default function YourComponent

choose one of the ways and u can talk with meir too about it to make code consistent  as possible